powerpacks.go

```diff
@@ -123,6 +126,41 @@ type NumberRemoveParams struct {
 	Unrent bool `json:"unrent,omitempty"`
 }
 
+type PowerpackAddNumberOptions struct {
+	//Service can be 'sms' or 'mms'. Default to 'sms' when not set.
+	Service string `json:"service,omitempty" url:"service,omitempty"`
+}
+
+type PowerpackFindNumberOptions struct {
+	//Service can be 'sms' or 'mms'. Default to 'sms' when not set.
+	Service string `json:"service,omitempty" url:"service,omitempty"`
+}
+
+type PowerpackAddTollfreeOptions struct {
+	//Service can be 'sms' or 'mms'. Default to 'sms' when not set.
+	Service string `json:"service,omitempty" url:"service,omitempty"`
+}
+
+type PowerpackListShortcodeOptions struct {
+	//Service can be 'sms' or 'mms'. Default to 'sms' when not set.
+	Service string `json:"service,omitempty" url:"service,omitempty"`
+}
+
+type PowerpackListTollfreeOptions struct {
+	//Service can be 'sms' or 'mms'. Default to 'sms' when not set.
```

Comment:
s/Default/Defaults

here and elsewhere

Reply:
changes made